backend/codesirius/problems/views/hidden_test.py

```
@@ -1,43 +1,126 @@
+            logger.info("Presigned POST URL generated successfully with 16MB limit")
+
+            return CodesiriusAPIResponse(
+                message="Presigned POST URL generated",
+                data={
+                    "url": presigned_post["url"],
+                    "fields": presigned_post["fields"],
+                },
+            )
+        except (Problem.DoesNotExist, PermissionDenied) as e:
+            raise ValidationError({"problem_id": e})
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)